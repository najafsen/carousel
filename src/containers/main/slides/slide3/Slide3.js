import React from "react";
import "./Slide3.scss";

export const Slide3 = () => (
  <div className="slide-content">
    <div className="slide3">
      <h1>
        These slides can contain anything, with any layout! Like previous
        slides.
      </h1>
    </div>
  </div>
);
