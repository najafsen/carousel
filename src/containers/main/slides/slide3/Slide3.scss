.slide3 {
  flex: 1;
  padding: 64px 104px;
  background-color: cadetblue;
  color: #ffffff;

  h1 {
    font-size: 72px;
  }
}
