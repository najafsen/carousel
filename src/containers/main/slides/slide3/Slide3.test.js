import React from "react";
import { render } from "@testing-library/react";
import { Slide3 } from "./Slide3";

describe("Slide3", () => {
  test("matches snapshot", () => {
    const { asFragment } = render(<Slide3 />);
    expect(asFragment()).toMatchSnapshot();
  });
});
