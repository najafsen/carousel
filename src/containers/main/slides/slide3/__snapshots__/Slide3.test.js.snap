// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slide3 matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="slide-content"
  >
    <div
      class="slide3"
    >
      <h1>
        These slides can contain anything, with any layout! Like previous slides.
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
